frontend/src/libs/components/router/router.tsx

```diff
@@ -30,7 +30,23 @@ const Router = (): JSX.Element => {
         element={<ProtectedRoute allowedUserGroup={UserGroupKey.BUSINESS} />}
       >
         <Route
-          path={AppRoute.DASHBOARD}
+          path={AppRoute.DASHBOARD_ORDERS}
+          element={
+            <PageLayout>
+              <Dashboard />
+            </PageLayout>
+          }
+        />
+        <Route
+          path={AppRoute.DASHBOARD_TRUCKS}
+          element={
+            <PageLayout>
+              <Dashboard />
+            </PageLayout>
+          }
+        />
+        <Route
+          path={AppRoute.DASHBOARD_DRIVERS}
```

Comment:
these routes should be protected - only business can view it

Reply:
These routes already are protected (<Route
        path={AppRoute.ROOT}
        element={<ProtectedRoute allowedUserGroup={UserGroupKey.BUSINESS} />}
      >)